exporter/signalfxexporter/translation/translator.go

```diff
@@ -266,6 +344,86 @@ func (mp *MetricTranslator) TranslateDimension(orig string) string {
 	return orig
 }
 
+// aggregateDatapoints aggregates datapoints assuming that they have
+// the same Timestamp, MetricType, Metric and Source fields.
+func aggregateDatapoints(
```

Comment:
Might be nice to have a unit test just for this fcn and getCommonDimensions but they're already covered, so totally up to you.

Reply:
Yea, I'd like to use the same test framework with defined input/output and get it covered through `TranslateDataPoints`